lib/Open/index.js

```diff
@@ -1,6 +1,7 @@
 const fs = require('graceful-fs');
 const directory = require('./directory');
 const Stream = require('stream');
+const axios = require('axios');
```

Comment:
As discussed, axios can not be part of this PR.   You can submit a second PR but please do axios as a new source instead of replacing functionality.  Backwards compatibility matters